src/main/java/org/commcare/formplayer/application/AbstractBaseController.java

```diff
@@ -131,7 +122,7 @@ private ExceptionResponseBean getPrettyExceptionResponse(Exception exception, Fo
     public ExceptionResponseBean handleHttpRequestError(FormplayerHttpRequest req, HttpClientErrorException exception) {
         incrementDatadogCounter(Constants.DATADOG_ERRORS_EXTERNAL_REQUEST, req);
         log.error(String.format("Exception %s making external request %s.", exception, req));
-        raven.sendRavenException(exception, Event.Level.INFO);
+        Sentry.captureException(exception);
```

Comment:
Is there a reason this doesn't go through FormplayerSentry?

Reply:
The only use case for `FormplayerSentry.captureException` is to alter the level that it is captured at. The default SDK always captures exceptions at the ERROR level but we have a number of places where we capture them at INFO or WARNING.

Previously we used the `FormplayerSentry` methods to apply all the available scope to the events as well but since that is now handled elsewhere there's no need to use it.

Reply:
Got it , thanks.